openpype/hosts/flame/plugins/publish/collect_test_selection.py

```diff
@@ -17,10 +16,40 @@ class CollectTestSelection(pyblish.api.ContextPlugin):
     hosts = ["flame"]
 
     def process(self, context):
-        self.log.info(opflame.selection)
+        self.log.info(
+            "Active Selection: {}".format(opfapi.CTX.selection))
 
-        sequence = lib.get_current_sequence(opflame.selection)
+        sequence = opfapi.get_current_sequence(opfapi.CTX.selection)
 
+        self.test_imprint_data(sequence)
+        self.test_otio_export(sequence)
+
+    def test_otio_export(self, sequence):
```

Comment:
What does this method do? It seems like it's creating file in user's dir to test if it works but it is not cleaned up afterwards. Also I would recommend using tempdir instead of user's dir if it doesn't matter.

Reply:
addressed in gitkraken://repolink/33c731ef76967f4bd3f6ba8b287ecc0c52412f33/commit/74a4fbfea60ff34866000a49962a26d40e6b9fd8?url=https%3A%2F%2Fgithub.com%2Fpypeclub%2FOpenPype.git